Dockerfile

```diff
@@ -2,9 +2,10 @@
 FROM golang:1.21 AS tinygo-llvm
 
 RUN apt-get update && \
-    apt-get install -y apt-utils make cmake clang-15 ninja-build
+    apt-get install -y apt-utils make cmake clang-15 ninja-build && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Suggestion: more savings:
```
    rm -rfv \
        /var/lib/apt/lists/* \
        /var/log/* \
        /var/tmp/* \
        /tmp/*
```

Reply:
Thanks for the improvement @b0ch3nski I have modified this PR accordingly.